canbench-bin/src/lib.rs

```diff
@@ -25,6 +25,7 @@ const BENCH_PREFIX: &str = "__canbench__";
 pub fn run_benchmarks(
     canister_wasm_path: &PathBuf,
     pattern: Option<String>,
+    init: Option<String>,
```

Comment:
To keep this method generic, would it make sense to pass a binary blob here and decode the hex string before this method is invoked?

```suggestion
    init_args: Option<Vec<u8>>
```

Reply:
I changed it to just `Vec<u8>` because then we can just fall back to `default` on parsing rather than keeping the `Option` around.